pyproject.toml

```diff
@@ -44,7 +45,7 @@ myst-parser = "^0.15.2"
 sphinx-autoapi = "^1.8.4"
 sphinx-rtd-theme = "^1.0.0"
 nautobot-chatops = "^1.5.1"
-nautobot-chatops-ipfabric = "^1.1.1"
+nautobot-chatops-ipfabric = "^2.0.0"
```

Comment:
this implicitly drops support for ipfabric <5.x.x. Should we mention it? (matrix)